client/extensions/woocommerce/state/sites/review-replies/handlers.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Internal dependencies
+ */
+import { dispatchRequest } from 'state/data-layer/wpcom-http/utils';
+import request from 'woocommerce/state/sites/http-request';
+import {
+	WOOCOMMERCE_REVIEW_REPLIES_UPDATED,
+	WOOCOMMERCE_REVIEW_REPLIES_REQUEST,
+} from 'woocommerce/state/action-types';
+
+export default {
+	[ WOOCOMMERCE_REVIEW_REPLIES_REQUEST ]: [ dispatchRequest(
+		handleReviewRepliesRequest,
+		handleReviewRepliesRequestSuccess,
+		handleReviewRepliesRequestError
+	) ],
+};
+
+export function handleReviewRepliesRequest( { dispatch }, action ) {
+	const { siteId, reviewId } = action;
+	dispatch( request( siteId, action, '/wp/v2' ).get( `comments?parent=${ reviewId }&per_page=50` ) );
```

Comment:
Why 50? The use case is typically 0 or 1 and occasionally 2, right? We don't intend on paginating these right? I guess 50 is fine but seems like overkill :)

Reply:
Yeah the use case is usually just a couple. I biggest a larger number for the reason you mentioned - not implementing pagination. Maybe something smaller like 15 is more reasonable.